src/Drifty_CLI.java

```diff
@@ -1,291 +1,192 @@
-                        fName = SC.nextLine();
+                        System.out.print(FILE_NAME_WITH_EXTENSION);
+                        fileName = SCANNER.nextLine();
                     }
                 } else {
-                    System.out.print("Enter the filename (with file extension) : ");
-                    fName = SC.nextLine();
+                    System.out.print(FILE_NAME_WITH_EXTENSION);
+                    fileName = SCANNER.nextLine();
                 }
             }
-            new FileDownloader(link, fName, downloadsFolder).run();
-            System.out.println("Press Q to Quit Or Press any Key to Continue");
-            String quit = SC.nextLine().toLowerCase();
-            if(quit.equals("q")){
-                logger.log("INFO", "Application Terminated!");
+            new FileDownloader(link, fileName, downloadsFolder).run();
+            System.out.println(QUIT_OR_CONTINUE);
+            String quit = SCANNER.nextLine().toLowerCase();
+            if (quit.equals("q")) {
+                logger.log(LOGGER_INFO, APPLICATION_TERMINATED);
                 break;
             }
-            printBanner();
+            DriftyUtility.printBanner();
         }
     }
 
     /**
      * This function takes a folder path as input from the user, where the file will be saved.
      */
-    private static void enterDownloadsFolder(){
-        System.out.print("Enter the directory in which you want to download the file : ");
-        downloadsFolder = SC.nextLine();
-        if (System.getProperty("os.name").contains("Windows")) {
-            downloadsFolder = SC.nextLine().replace('/', '\\');
+    private static void enterDownloadsFolder() {
+        System.out.print(DIRECTORY_TO_DOWNLOAD_FILE);
+        downloadsFolder = SCANNER.nextLine();
+        if (System.getProperty(OS_NAME).contains(WINDOWS_OS_NAME)) {
+            downloadsFolder = SCANNER.nextLine().replace('/', '\\');
             if (!(downloadsFolder.endsWith("\\"))) {
                 downloadsFolder = downloadsFolder + System.getProperty("file.separator");
             }
         }
-        logger.log("INFO", "Custom Directory Entered : " + downloadsFolder);
+        logger.log(LOGGER_INFO, "Custom Directory Entered : " + downloadsFolder);
     }
 
     /**
      * This function tries to detect the default downloads folder and save the file in that folder
      */
-    private static void saveToDefault(){
-        System.out.println("Trying to auto-detect default Downloads folder...");
-        logger.log("INFO", "Trying to auto-detect default Downloads folder...");
-        if (!System.getProperty("os.name").contains("Windows")) {
-            String home = System.getProperty("user.home");
-            downloadsFolder = home + "/Downloads/";
-        }
-        else {
+    private static void saveToDefault() {
+        System.out.println(TRYING_TO_AUTOMATICALLY_DETECT_FILE);
```

Comment:
Please change `TRYING_TO_AUTOMATICALLY_DETECT_FILE` to `TRYING_TO_AUTO_DETECT_FILE` to reduce the name of the constant.
```suggestion
        System.out.println(TRYING_TO_AUTO_DETECT_FILE);
```